src/datasource.ts

```diff
@@ -544,9 +544,7 @@ export class OpenSearchDatasource extends DataSourceWithBackend<OpenSearchQuery,
           (request.app === CoreApp.Explore &&
             target.queryType === QueryType.PPL &&
             (target.format === 'logs' || target.format === 'table')) ||
-          (request.app === CoreApp.Explore &&
-            target.luceneQueryType === LuceneQueryType.Traces &&
-            getTraceIdFromLuceneQueryString(target.query ?? ''))
```

Comment:
do we not care about the getTraceId part of the if condition anymore?

Reply:
We want all trace queries to go through the backend, whether they're trace list or single trace. 
Tbh it looks like even before this, all trace queries in Explore went through the backend because of the `metrics` field above (trace queries have a metric field "count", by default 😔 )